biotracks/readfile.py

```diff
@@ -5,28 +5,12 @@
 
 import pandas as pd
 import xlrd
-from xlrd import XLRDError
 
 from .names import (
     X_COORD_NAME, Y_COORD_NAME, FRAME_NAME, OBJECT_NAME, LINK_NAME
 )
 
 
-def xls_to_csv(xls_file):
-    """Utility function to read Excel files."""
-
-    x = xlrd.open_workbook(xls_file)
-    x1 = x.sheet_by_index(0)
-    name, extension = os.path.splitext(xls_file)
-
-    with open(name + '.csv', 'w', encoding='utf-8') as csv_file:
-        writecsv = csv.writer(csv_file, quoting=csv.QUOTE_NONE)
-        for rownum in range(x1.nrows):
-            writecsv.writerow(x1.row_values(rownum))
-
-    csv_file.close()
-
-
```

Comment:
It's OK to remove this code now, but we need to keep in mind we might need an Excel reader in the future, and therefore perhaps something like this back again.